src/docs/framework-integration/angular.md

```diff
@@ -39,211 +43,179 @@ top-most-directory/
+Install the `@stencil/angular-output-target` dependency to your Stencil component library package.
 
-`npm i @stencil/angular-output-target`
-
-or
-
-`yarn add @stencil/angular-output-target`
+```bash
+npm install @stencil/angular-output-target --save-dev
+```
 
-### Add the Angular Wrapper Function to your Stencil Component Library
-With the Angular Output Target package installed, we can now configure our Stencil Component Library to build our Angular wrapped components. In the `stencil.config.ts` file of your Stencil component library, add the Angular wrapper function. 
+In your project's `stencil.config.ts`, add the `angularOutputTarget` configuration to the `outputTargets` array:
 
-Within your Stencil’s config file, you can import the Angular wrapper function for use within the outputTargets array. If you copy and paste this, ensure you update the “my-workspace” and “my-lib” to coincide with what you named your component library, as well as to update the `componentCorePackage` to the name of your Stencil component library.
+```ts
+import { angularOutputTarget } from '@stencil/angular-output-target';
 
-```tsx
-import { angularOutputTarget as angular } from '@stencil/angular-output-target';
 export const config: Config = {
-  namespace: 'demo',
+  namespace: '',
   outputTargets: [
-    angular({
-      componentCorePackage: `your-stencil-library-name`,
-      directivesProxyFile: `../your-angular-workspace-name/projects/your-angular-library-name/src/lib/stencil-generated/components.ts`
+    angularOutputTarget({
+      componentCorePackage: 'your-stencil-library-package-name',
+      directivesProxyFile: '../angular-workspace/projects/component-library/src/lib/stencil-generated/components.ts',
     }),
-    {
-      type: 'dist',
-      esmLoaderPath: '../loader',
-    },
-    {
-      type: 'dist-custom-elements',
-    },
   ],
 };
 ```
 
-Once you have added this and have `directivesProxyFile` pointing to the correct directory of your Angular component library, you can build Stencil. 
+> The `directivesProxyFile` is the relative path to the file that will be generated with all of the Angular component wrappers. You will replace the file path to match your project's structure and respective names. You can generate any file name instead of `components.ts`.
 
-`npm run build`
+You can now build your Stencil component library to generate the component wrappers.
 
-Or
+```bash
+npm run build
+```
 
-`yarn build`
+If the build is successful, you will now have contents in the file specified in `directivesProxyFile`.
 
-Once the build is complete, you will see new files in your Angular component library's directory!
+You can now finally import and export the generated component wrappers for your component library. For example, in your main `AppModule` you may do the following:
 
+```ts
+import { COMPONENTS } from './stencil-generated/components';
```

Comment:
This may be a bit confusing to folks. I totally see what you're going for here - import the components from this path and add them to your declarations & exports fields.  However `./stencil-generated/components` also has this line:
```
import { Components } from 'stencil-library';
```
where I wonder if folks may think we've just made a typo with the casing of 'components' here. Does it make sense if we treat this guide more like a tutorial and give move concrete names here?
```
import { MyComponent } from './stencil-generated/components';

@NgModule({
  declarations: [
    ComponentLibraryComponent,
    MyComponent
  ],
  imports: [
  ],
  exports: [
    ComponentLibraryComponent
  ]
})
```

Reply:
The output targets generates an exported constant called `COMPONENTS` that is an array of all the component class declarations.

If you are creating a library, you would likely make use of this constant. Otherwise you would need to individually import and export all generated component classes (which is prone to errors when a developer deletes a deprecated component from Stencil, but forgets to update their Angular module). 

Open to suggestions here on how to best explain this is a generated constant value available to you. I'm not a huge fan of us calling the directory `stencil-generated` in our examples. I think including `stencil-` confuses the scope.

Reply:
Hmm I think I'm missing something then. I can't find a `COMPONENTS` constant in my project.  I've run through generating a new stencil component and angular libraries, with the directions in this PR. Is there somewhere specific I should be looking?

![Screen Shot 2022-03-01 at 1 29 00 PM](https://user-images.githubusercontent.com/1930213/156227807-07864fcc-7762-4436-adc6-83beb502e27c.png)


I looked in the output targets repo for 'COMPONENTS' (whole word, case sensitive) and couldn't find it either 🤔

Reply:
Great catch! I was getting my wires crossed with my output targets PR 🤦  we have an option that allows generating a constant called `DIRECTIVES`: https://github.com/ionic-team/stencil-ds-output-targets/blob/main/packages/angular-output-target/src/generate-angular-directives-file.ts#L28

Which is the incorrect label for the values. In my output targets refactor, I correct this name to align with the Angular community's naming convention.

I will need to update this PR to reference that variable name in the interim though.